man/tmpfiles.d.xml

```diff
@@ -282,14 +282,16 @@ L     /tmp/foobar -    -    -     -   /dev/null</programlisting>
 
         <varlistentry>
           <term><varname>C</varname></term>
-          <listitem><para>Recursively copy a file or directory, if the
-          destination files or directories do not exist yet or the
-          destination directory is empty. Note that this command will not
-          descend into subdirectories if the destination directory already
-          exists and is not empty. Instead, the entire copy operation is
+          <listitem><para>Recursively copy a file or directory, if the destination files or directories do
+          not exist yet or the destination directory is empty. The source file or directory to copy is
+          specified in the argument column. If the source path specified is relative it's considered relative
+          to the destination path. Note that this command will not descend into subdirectories if the
+          destination directory already exists and is not empty. Instead, the entire copy operation is
           skipped. If the argument is omitted, files from the source directory
-          <filename>/usr/share/factory/</filename> with the same name
-          are copied. Does not follow symlinks.</para></listitem>
+          <filename>/usr/share/factory/</filename> with the same name are copied. Note that the source file
+          paths are not affected by the <option>--root=</option> command line switch of
+          <command>systemd-tmpfiles</command>; use relative source paths for copying files from the tree
+          specified with <option>--root=</option>. Does not follow symlinks.</para></listitem>
```

Comment:
Contrary to `L`, `C` indeed does ignore `--root`, or speaking more precisely, it takes the source file from outside of `--root`. This behaviour is different than for other config types. As I wrote above, this makes little sense because paths in configuration files under `--root` cannot meaningfully refer to files outside of the tree. And when `systemd-tmpfiles` is then run during normal operation, the effect of those operations would be different. If the files are volatile and are copied again, the result will be different. So this violates the rule that `systemd-tmpfiles --create` is idempotent.

Instead of documenting what I consider a bug, I think `C` should be made to obey `--root` instead. If people want to copy file from outside of the tree into the tree, they should use rsync. Mixing the configuration from the tree with files from outside of the tree is just too ugly.